runtime/queries/zig/highlights.scm

```diff
@@ -0,0 +1,191 @@
+
+; Zig
+
+; Variables
+; --------------
+(identifier) @variable
+
+(parameter (identifier) @variable)
+
+; ((identifier) @constant
+;  (#vim-match? @constant "^[A-Z][A-Z\\d_]+$'"))
+
+; function definition
+(function_signature
+  name: (identifier) @function)
+
+(function_declaration
+  name: (identifier) @function)
+
+; Function calls
+(call_expression
+  function: (identifier) @function)
+
+(build_in_call_expr
+  function: (identifier) @function.builtin
+)
+
+;; other identifiers
+(type_identifier) @type
+(primitive_type) @type.builtin
+(field_identifier) @field
```

Comment:
`field` is not a scope we use, it should be `property`.

Reply:
How do I know which scopes are available? I have no idea where these come from.

Reply:
Ah, it's listed here: https://docs.helix-editor.com/themes.html